clang/test/Driver/woa-lto.c

```diff
@@ -0,0 +1,15 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
"woa-" is not a clear filename prefix. Suggest `windows-lto.c`

In general, only -### should be used in driver tests to suppress action running. We can then drop `REQUIRES: `.